data/sql/updates/db_world/2020_04_21_00.sql

```diff
@@ -0,0 +1,30 @@
+-- DB update 2020_04_17_02 -> 2020_04_21_00
+DROP PROCEDURE IF EXISTS `updateDb`;
+DELIMITER //
+CREATE PROCEDURE updateDb ()
+proc:BEGIN DECLARE OK VARCHAR(100) DEFAULT 'FALSE';
+SELECT COUNT(*) INTO @COLEXISTS
+FROM information_schema.COLUMNS
+WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_world' AND COLUMN_NAME = '2020_04_17_02';
+IF @COLEXISTS = 0 THEN LEAVE proc; END IF;
+START TRANSACTION;
+ALTER TABLE version_db_world CHANGE COLUMN 2020_04_17_02 2020_04_21_00 bit;
+SELECT sql_rev INTO OK FROM version_db_world WHERE sql_rev = '1518477962308987200'; IF OK <> 'FALSE' THEN LEAVE proc; END IF;
+--
+-- START UPDATING QUERIES
+--
+
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1518477962308987200');
+
+DELETE FROM `item_loot_template` WHERE `Entry` = 54537 AND `Item` = 49426;
+INSERT INTO `item_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(54537, 49426, 0, 100, 0, 1, 0, 2, 2, '');
```

Comment:
how is this related to `g3dlite` and why this is not using the pending import? I guess it's a mistake?

Reply:
i think its merge shit >_< w8 few minutes